src/core/tests/type_prop/inverse.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/inverse.hpp"
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/test_assertions.hpp"
+#include "common_test_utils/type_prop.hpp"
+#include "openvino/op/constant.hpp"
+
+using namespace testing;
+
+class TypePropInverseV14Test : public TypePropOpTest<ov::op::v14::Inverse> {};
+
+TEST_F(TypePropInverseV14Test, default_ctor) {
+    const auto data = ov::op::v0::Constant::create(ov::element::f64, ov::Shape{2, 2}, {1.0f, 1.0f, 1.0f, 1.0f});
+    const auto op = make_op();
+    op->set_arguments(ov::OutputVector{data});
+    op->validate_and_infer_types();
+
+    EXPECT_EQ(op->get_input_size(), 1);
+    EXPECT_EQ(op->get_output_size(), 1);
+    EXPECT_EQ(op->get_output_element_type(0), ov::element::f64);
+    EXPECT_EQ(op->get_output_partial_shape(0), ov::PartialShape({2, 2}));
+}
+
+TEST_F(TypePropInverseV14Test, input_f64_2x2_constant) {
+    const auto data = ov::op::v0::Constant::create(ov::element::f64, ov::Shape{2, 2}, {1.0f, 1.0f, 1.0f, 1.0f});
+    const auto op = make_op(data, false);
+    EXPECT_EQ(op->get_element_type(), ov::element::f64);
+    EXPECT_EQ(op->get_output_partial_shape(0), (ov::PartialShape{2, 2}));
+}
+
+TEST_F(TypePropInverseV14Test, input_f32_static) {
```

Comment:
It would be nice to add couple label tests here, but looking on code it's very simple case so probably this can be done in separate PR, maybe doublecheck manually just to be sure

Reply:
Confirmed, it works, will add it after CF